lib/fizzy/execute.cpp

```diff
@@ -408,7 +440,7 @@ inline T fmin(T a, T b) noexcept
     if (std::isnan(a) || std::isnan(b))
         return std::numeric_limits<T>::quiet_NaN();  // Positive canonical NaN.
 
-    if (a == 0 && b == 0 && (std::signbit(a) == 1 || std::signbit(b) == 1))
```

Comment:
@chfast [`std::signbit`](https://en.cppreference.com/w/cpp/numeric/math/signbit) actually returns a bool, why did use integer literals here and below?

Reply:
I don't remember. Probably it is more clear that signbit is `1`, while the meaning of `true` bit is confusing.